src/components/electricity_tariffs/energycharts/electricity_tariff.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<div class="electricity-tariff-energycharts">
+		<openwb-base-alert subtype="info">
+			Börsenstrompreise von Energy Charts (energy-charts.info)<br />
+			Die Rohdaten werden von Wissenschaftlern des Fraunhofer-Institut
+			für Solare Energiesysteme ISE aus zahlreichen Quellen stündlich
+			oder täglich abgerufen und für die Darstellung aufbereitet.
+		</openwb-base-alert>
+		<openwb-base-select-input
+			title="Land"
+			notSelected="Bitte auswählen"
+			:options="[
+				{ value: 'DE-LU', text: 'Deutschland' },
+				{ value: 'AT', text: 'Österreich' },
+				{ value: 'CH', text: 'Schweiz' },
+			]"
+			:model-value="electricityTariff.configuration.country"
+			@update:model-value="
+				updateConfiguration($event, 'configuration.country')
+			"
+		>
+		</openwb-base-select-input>
```

Comment:
Elemente ohne Inhalt können direkt als solche gekennzeichnet werden.
```suggestion
		/>
```

Reply:
Done